tests/src/test/java/alluxio/master/JournalShutdownIntegrationTest.java

```diff
@@ -113,6 +117,9 @@ public void run() {
   public final void after() throws Exception {
     mExecutorsForClient.shutdown();
     ConfigurationTestUtils.resetConfiguration();
+    RetryHandlingBlockWorkerClientTestUtils.reset();
```

Comment:
if these all need to be reset whenever the system is used, we should probably consolidate them into one method. We don't need to make that a part of this PR though